### PR TITLE
#213 Unpublish Command

### PR DESCRIPTION
**Pull request by JohnPTobe**
Adding a command to remove images from a remote registry per issue #213 
Also updating several commands to infer an image name from a manifest either in the current directory or a specified manifest file per issue #199.
Also updated build output to give example run and publish commands when finished per issue #200.
Finally, updates were made to gitlab ci for pipeline builds on high side.

### CODE REVIEW

File: commands/unpublish.go [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+}
+
+//PrintPublishUsage prints the seed publish usage information, then exits the program
+func PrintUnpublishUsage() {
+	util.PrintUtil("\nUsage:\tseed unpublish [-in IMAGE_NAME] [-M MANIFEST] [-v VERSION] [-r REGISTRY_NAME] [-o ORG_NAME] [-u username] [-p password] [Conflict Options]\n")
+	util.PrintUtil("\nAllows for the removal of seed compliant images from a registry.\n")
+	util.PrintUtil("\nOptions:\n")
+	util.PrintUtil("  -%s -%s Docker image name to publish\n",
+		constants.ShortImgNameFlag, constants.ImgNameFlag)
+	util.PrintUtil("  -%s -%s\t  Manifest file to use if an image name is not specified.(default is seed.manifest.json within the current directory)\n",
```
Comment (emimaesmith) [inline commands/unpublish.go]: Copypasta from above 🍝

File: constants/constants.go [hunk 2]
```diff
@@ -38,7 +39,7 @@ const ShortDockerfileFlag = "D"
 const ManifestFlag = "manifest"
 
 //ShortManifestFlag defines the shorthand manifest file
-const ShortManifestFlag = "m"
+const ShortManifestFlag = "M"
```
Comment (emimaesmith) [inline constants/constants.go]: Is this to differentiate between mount and manifest?

File: main.go [hunk 3]
```diff
@@ -798,15 +880,16 @@ func DefineFlags() {
...
 	util.PrintUtil("  build \tBuilds Seed compliant Docker image\n")
 	util.PrintUtil("  batch \tExecutes Seed compliant docker image over multiple iterations\n")
 	util.PrintUtil("  init  \tInitialize new project with example seed.manifest.json file\n")
-	util.PrintUtil("  list  \tAllows for listing of all Seed compliant images residing on the local system\n")
-	util.PrintUtil("  publish\tAllows for publish of Seed compliant images to remote Docker registry\n")
-	util.PrintUtil("  pull\t\tAllows for pulling Seed compliant images from remote Docker registry\n")
-	util.PrintUtil("  run   \tExecutes Seed compliant Docker docker image\n")
+	util.PrintUtil("  list  \tLists all Seed compliant images residing on the local system\n")
+	util.PrintUtil("  publish\tPublishes Seed compliant images to remote Docker registry\n")
+	util.PrintUtil("  unpublish\tRemoves images from remote Docker registry\n")
```
Comment (emimaesmith) [inline main.go]: We had these commands listed in alphabetical order. I'd suggest moving unpublish to the bottom to maintain that order.

File: constants/constants.go (hunk 2 above)
Comment (JohnPTobe) [reply constants/constants.go]: Yes, batch and run now use both so they need to be different.

**Comment by JohnPTobe** [comment]
Fixed comments and tests pass

**Pull request merged**